packages/devtools_app/lib/src/memory/memory_filter.dart

```diff
@@ -158,8 +158,6 @@ class SnapshotFilterState extends State<SnapshotFilterDialog>
     for (var key in oldFiltered.keys) {
       oldFilteredLibraries[key] = oldFiltered[key].first.hide;
     }
-
-    cancel();
```

Comment:
The listeners this was canceling were removed in https://github.com/flutter/devtools/pull/2275, so this should be safe to remove.